synapse/config/experimental.py

```diff
@@ -32,3 +32,6 @@ def read_config(self, config: JsonDict, **kwargs):
 
         # MSC2716 (backfill existing history)
         self.msc2716_enabled = experimental.get("msc2716_enabled", False)  # type: bool
+
+        # MSC3266 (room summary api)
+        self.mxc3266_enabled = experimental.get("mxc3266_enabled", False)  # type: bool
```

Comment:
```suggestion
        self.msc3266_enabled = experimental.get("msc3266_enabled", False)  # type: bool
```

Reply:
Yeah hah I hit this locally and just forgot to push, thanks!